pkgs/applications/editors/vscode/default.nix

```diff
@@ -22,6 +22,7 @@ let
   rpath = lib.concatStringsSep ":" [
     atomEnv.libPath
     "${lib.makeLibraryPath [gtk2]}"
+    "${lib.makeLibraryPath [at_spi2_atk]}"
```

Comment:
This seems to be necessary for a correct RPATH patch? If I don't add it there, ldd outputs multiple libatk-bridge-2.0.so.0 entries.